index.html

```diff
@@ -21,51 +21,130 @@
                 <div class="ic_icon">
                     <img src="assets/shapes/ic_menu.jpg" alt="">
                 </div>
+                <ul class="nav_list_container">
+                    <li><a href="#Home">Home</a></li>
+                    <li><a href="#Portfolio">Portfolio</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Contact">contact</a></li>
+                </ul>
             </div>
         </nav>
     </header>
     <main>
         <section class="headline-section">
-            <div class="header-text">
-                <h1>Charlie</h1>
-                <h1>Giorgania</h1>
-            </div>
-            <div class="header-description">
-                <p>Hello! I am a software developer! I can help
-                    you build a product, feature or website. Take a look at my work.
-                </p>
-                <p>If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-            </div>
-            <div class="headline-button">
-                <a href="#headline-button">Start Collaboration</a>
+            <div class="left-col">
+                <div class="text-container">
+                    <div class="header-text">
+                        <h1>Charlie</h1>
+                        <h1>Giorgania</h1>
+                    </div>
+                    <div class="header-description">
+                        <p>Hello! I am a software developer! I can help
+                            you build a product, feature or website. Take a look at my work.
+                        </p>
+                        <p>If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                    </div>
+                    <div class="headline-button">
+                        <a href="#headline-button">Start Collaboration</a>
+                    </div>
+                    <div class="shape2"></div>
+                </div> 
             </div>
-            <div class="shape2"></div>
+            <div class="right-col"></div>
+
+            
         </section>
         <section id="projects">
-            <h1>Projects</h1>
+            <div class="desktop-project-designs">
```

Comment:
Dear @Mylo16, Please consider reducing the large space between the `heading` and `projects` sections. This is to make sure your design looks like the one on the figma design.

![Screenshot 2023-02-24 180321](https://user-images.githubusercontent.com/69077061/221211792-b3839cb6-8c8c-4900-844f-0cfc32eeb13d.png)